qcodes/utils/validators.py

```diff
@@ -189,6 +192,8 @@ def __repr__(self) -> str:
 
 class Numbers(Validator):
     """
+    Requires a number  of type int, float, numpy.integer or numpy.floating.
+
     Args:
         min_value: Minimal value allowed, default -inf.
         max_value: Maximal value allowed, default inf.
```

Comment:
Could you fix the todo below too?

Reply:
OK

Reply:
Thanks :)

Reply:
Fixed the docs to included `Raises` in all the methods.